### PR TITLE
LP Transformation tests decrease thresholds

### PR DESCRIPTION
**Pull request by evkotov**
### Details:
 - remove thresholds from SetUp() test classes
 - use rel_threshold = 1.1, abs_threshold = 1.0e-4 defined in LayerTransformation class as default values

### Tickets:
 - 131727

### CODE REVIEW

File: src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp [hunk 1]
```diff
@@ -36,9 +36,6 @@ std::string ConvolutionTransformation::getTestCaseName(const testing::TestParamI
 }
 
 void ConvolutionTransformation::SetUp() {
-    rel_threshold = 1.0e+10;
```
Comment (eshoguli) [inline src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp:39]: can you explore this case in more details? I think something wrong here.

Comment (eshoguli) [reply src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp:39]: my mistake

**Review by eshoguli — APPROVED** [review]
LGTM

File: src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp (hunk 1 above)
Comment (itikhono) [reply src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp:39]: resolved

**Pull request merged**